cypress/support/Pages/allMethodsList.js

```diff
@@ -15,8 +15,15 @@ class MethodsPage {
 
```

Comment:
do czego ten plik allMethodsList?

Reply:
jest to klasa dla strony https://example.cypress.io/ - na której znajdują się wszystkie metody opisane na sinku więc "allMethodsList"